src/components/@shared/VerifiedBadge/index.tsx

```diff
@@ -48,6 +55,32 @@ export default function VerifiedBadge({
           <span>{text}</span>
         </div>
         <div className={styles.details}>
+          {process.env.NEXT_PUBLIC_BADGE_COMPLIANCE_LABELS_ENABLE === 'true' &&
+            complianceTypes && (
+              <ul style={{ listStyle: 'none', padding: 0 }}>
+                {complianceTypes.map((complianceType) => {
+                  const complianceCaption =
+                    ComplianceTypeLookup.getCaption(complianceType)
+
+                  return (
+                    <li
+                      key={complianceCaption}
+                      style={{ display: 'flex', alignItems: 'center' }}
+                    >
+                      <Checkmark
+                        style={{
+                          width: '24px',
+                          height: '24px',
+                          marginRight: '0.1rem'
+                        }}
+                      />
```

Comment:
issue: would prefer styles to be handled by css module file, to be in line with the rest of the portal code

Reply:
@moritzkirstein 
Did you pull the latest changes and restarted?
I just tried it on a clean repo clone and it just works. I can set in my .env
```
NEXT_PUBLIC_CATALOG_FILTER_COMPLIANCE_GAIA_X_ENABLE=true
```
(Not NEXT_PUBLIC_SERVICE_SD_COMPLIANCE_GAIA_X_ENABLE, that is to mark the asset as compliant during publishing.)
A few moments later the page enables the filter, I don't need to reload the page.

Reply:
Okay thanks for the input, I will check this again.